src/ResourceManager/Network/AzureRM.Network.psd1

```diff
@@ -290,7 +290,8 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Get-AzureRmVirtualNetwork', 'New-AzureRmVirtualNetwork', 
                'Get-AzureRmVirtualNetworkGatewayBgpPeerStatus', 
                'Get-AzureRmVirtualNetworkGatewayAdvertisedRoute', 
-               'Get-AzureRmVirtualNetworkGatewayLearnedRoute'
+               'Get-AzureRmVirtualNetworkGatewayLearnedRoute',
+               'Get-AzureRmNetworkUsage'
```

Comment:
Added manually for now. [related discussion](https://github.com/EvgenyAgafonchikov/azure-powershell/commit/b810a9e0ef358449c00b313c9f3b224d2111178b#commitcomment-22031581)